packaging/RPMS/Fedora/rabbitmq-server.spec

```diff
@@ -188,7 +190,9 @@ fi
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq
 %attr(0750, rabbitmq, rabbitmq) %dir %{_localstatedir}/lib/rabbitmq/mnesia
 %attr(0755, rabbitmq, rabbitmq) %dir %{_localstatedir}/log/rabbitmq
-%dir %{_sysconfdir}/rabbitmq
+%attr(2755, -, -) %dir %{_sysconfdir}/rabbitmq
```

Comment:
@dumbbell I am not totally sure about this `attr(2755, -, -)`, even if it works correctly:
```
ls -la /etc/rabbitmq/
total 12
drwxr-sr-x.  2 root rabbitmq 4096 Jun 22 12:41 .
drwxr-xr-x. 80 root root     4096 Jun 22 12:40 ..
-rw-r-----.  1 root rabbitmq   23 Jun 22 12:41 enabled_plugins
```
WDYT?